03:50 31-07-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


04:00 31-07-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


04:30 31-07-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


05:00 31-07-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


05:30 31-07-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


06:00 31-07-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


06:30 31-07-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


07:00 31-07-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


07:30 31-07-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


08:00 31-07-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


08:30 31-07-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


09:00 31-07-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


09:30 31-07-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


10:00 31-07-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


10:30 31-07-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


11:00 31-07-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


11:30 31-07-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


12:00 31-07-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


12:30 31-07-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


13:00 31-07-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


13:30 31-07-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


14:00 31-07-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


14:30 31-07-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


15:00 31-07-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


15:30 31-07-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


16:00 31-07-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


16:30 31-07-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


17:00 31-07-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


17:30 31-07-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


18:00 31-07-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


18:32 31-07-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


19:00 31-07-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


19:30 31-07-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


20:00 31-07-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


20:30 31-07-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


21:00 31-07-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


21:30 31-07-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


22:00 31-07-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


22:30 31-07-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


23:00 31-07-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


23:30 31-07-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


00:00 01-08-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


00:30 01-08-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


01:00 01-08-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


01:30 01-08-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


02:00 01-08-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


02:30 01-08-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


03:00 01-08-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


03:30 01-08-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


04:00 01-08-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


04:30 01-08-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


05:00 01-08-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


05:30 01-08-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


06:00 01-08-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


06:31 01-08-2025

# Get in right directory

# Get up to date
fatal: unable to access 'https://github.com/LaurensGamet/Somtoday_Agendas.git/': Could not resolve host: github.com

# Run main file
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/urllib3/connection.py", line 169, in _new_conn
    conn = connection.create_connection(
  File "/usr/lib/python3/dist-packages/urllib3/util/connection.py", line 73, in create_connection
    for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):
  File "/usr/lib/python3.9/socket.py", line 953, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno -3] Temporary failure in name resolution

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/urllib3/connectionpool.py", line 700, in urlopen
    httplib_response = self._make_request(
  File "/usr/lib/python3/dist-packages/urllib3/connectionpool.py", line 383, in _make_request
    self._validate_conn(conn)
  File "/usr/lib/python3/dist-packages/urllib3/connectionpool.py", line 1017, in _validate_conn
    conn.connect()
  File "/usr/lib/python3/dist-packages/urllib3/connection.py", line 353, in connect
    conn = self._new_conn()
  File "/usr/lib/python3/dist-packages/urllib3/connection.py", line 181, in _new_conn
    raise NewConnectionError(
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPSConnection object at 0xb5e9d250>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/requests/adapters.py", line 439, in send
    resp = conn.urlopen(
  File "/usr/lib/python3/dist-packages/urllib3/connectionpool.py", line 756, in urlopen
    retries = retries.increment(
  File "/usr/lib/python3/dist-packages/urllib3/util/retry.py", line 576, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='api.somtoday.nl', port=443): Max retries exceeded with url: /rest/v1/icalendar/stream/58405be0-5611-4aba-be66-9894a1009f12/c5490491-11eb-4aca-a8f1-ea3f58e75d54 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0xb5e9d250>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/laurens/Somtoday_Agendas/Madelief/main.py", line 22, in <module>
    r = requests.get(url, allow_redirects=True)
  File "/usr/lib/python3/dist-packages/requests/api.py", line 76, in get
    return request('get', url, params=params, **kwargs)
  File "/usr/lib/python3/dist-packages/requests/api.py", line 61, in request
    return session.request(method=method, url=url, **kwargs)
  File "/usr/lib/python3/dist-packages/requests/sessions.py", line 542, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/lib/python3/dist-packages/requests/sessions.py", line 655, in send
    r = adapter.send(request, **kwargs)
  File "/usr/lib/python3/dist-packages/requests/adapters.py", line 516, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPSConnectionPool(host='api.somtoday.nl', port=443): Max retries exceeded with url: /rest/v1/icalendar/stream/58405be0-5611-4aba-be66-9894a1009f12/c5490491-11eb-4aca-a8f1-ea3f58e75d54 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0xb5e9d250>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
Done


06:51 01-08-2025

# Get in right directory

# Get up to date
fatal: unable to access 'https://github.com/LaurensGamet/Somtoday_Agendas.git/': Could not resolve host: github.com

# Run main file
The file does not exist
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/urllib3/connection.py", line 169, in _new_conn
    conn = connection.create_connection(
  File "/usr/lib/python3/dist-packages/urllib3/util/connection.py", line 73, in create_connection
    for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):
  File "/usr/lib/python3.9/socket.py", line 953, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno -3] Temporary failure in name resolution

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/urllib3/connectionpool.py", line 700, in urlopen
    httplib_response = self._make_request(
  File "/usr/lib/python3/dist-packages/urllib3/connectionpool.py", line 383, in _make_request
    self._validate_conn(conn)
  File "/usr/lib/python3/dist-packages/urllib3/connectionpool.py", line 1017, in _validate_conn
    conn.connect()
  File "/usr/lib/python3/dist-packages/urllib3/connection.py", line 353, in connect
    conn = self._new_conn()
  File "/usr/lib/python3/dist-packages/urllib3/connection.py", line 181, in _new_conn
    raise NewConnectionError(
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPSConnection object at 0xb5f20250>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/requests/adapters.py", line 439, in send
    resp = conn.urlopen(
  File "/usr/lib/python3/dist-packages/urllib3/connectionpool.py", line 756, in urlopen
    retries = retries.increment(
  File "/usr/lib/python3/dist-packages/urllib3/util/retry.py", line 576, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='api.somtoday.nl', port=443): Max retries exceeded with url: /rest/v1/icalendar/stream/58405be0-5611-4aba-be66-9894a1009f12/c5490491-11eb-4aca-a8f1-ea3f58e75d54 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0xb5f20250>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/laurens/Somtoday_Agendas/Madelief/main.py", line 22, in <module>
    r = requests.get(url, allow_redirects=True)
  File "/usr/lib/python3/dist-packages/requests/api.py", line 76, in get
    return request('get', url, params=params, **kwargs)
  File "/usr/lib/python3/dist-packages/requests/api.py", line 61, in request
    return session.request(method=method, url=url, **kwargs)
  File "/usr/lib/python3/dist-packages/requests/sessions.py", line 542, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/lib/python3/dist-packages/requests/sessions.py", line 655, in send
    r = adapter.send(request, **kwargs)
  File "/usr/lib/python3/dist-packages/requests/adapters.py", line 516, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPSConnectionPool(host='api.somtoday.nl', port=443): Max retries exceeded with url: /rest/v1/icalendar/stream/58405be0-5611-4aba-be66-9894a1009f12/c5490491-11eb-4aca-a8f1-ea3f58e75d54 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0xb5f20250>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
Done


07:00 01-08-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
The file does not exist
Done


07:30 01-08-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


08:00 01-08-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


08:30 01-08-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


09:00 01-08-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


09:30 01-08-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


10:00 01-08-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


10:30 01-08-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


11:00 01-08-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


11:30 01-08-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


12:00 01-08-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


12:30 01-08-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


13:00 01-08-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


13:30 01-08-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


14:00 01-08-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


14:30 01-08-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


15:00 01-08-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


15:30 01-08-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


16:00 01-08-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


16:30 01-08-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


17:00 01-08-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


17:30 01-08-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


18:00 01-08-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


18:30 01-08-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


19:00 01-08-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


19:30 01-08-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


20:00 01-08-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


20:30 01-08-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


21:00 01-08-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


21:30 01-08-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


22:00 01-08-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


22:30 01-08-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


23:00 01-08-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


23:30 01-08-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


00:01 02-08-2025

# Get in right directory

# Get up to date
warning: fetch updated the current branch head.
fast-forwarding your working tree from
commit f5e0317efe260e6e1ec34798fd38c6880e8901ff.
error: Your local changes to the following files would be overwritten by merge:
	Logs/Laurens.log
	Logs/Madelief.log
Please commit your changes or stash them before you merge.
Aborting
fatal: Cannot fast-forward your working tree.
After making sure that you saved anything precious from
$ git diff f5e0317efe260e6e1ec34798fd38c6880e8901ff
output, run
$ git reset --hard
to recover.

# Run main file
Done


00:30 02-08-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


01:00 02-08-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


01:30 02-08-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


02:00 02-08-2025

# Get in right directory

# Get up to date
Already up to date.

# Run main file
Done


